bundles/org.openhab.transform.jinja/README.md

```diff
@@ -1,9 +1,9 @@
 # Jinja Transformation Service
 
-Transforms a value using a jinja template.
+Transforms a value using a Jinja template.
 
-The main purpose of this transformer is the use in the home assistant discovery. Therfore not all features of the home assistant templating are supported.
-Basically on [Processing incoming data](https://www.home-assistant.io/docs/configuration/templating/#processing-incoming-data)
+The main purpose of this transformer is use in Home Assistant discovery. Therefore, not all features of the Home Assistant templating engine are supported.
```

Comment:
I'll use a separate PR to remove this paragraph, since it's no longer true (both that Home Assistant binding doesn't use this transform anymore, and also it supports most Home Assistant extensions to Jinja now)

Reply:
Good, i'll leave this in here and let's see if this or your PR gets merged first and rebase at that moment.